js/test/Ice/proxy/Client.ts

```diff
@@ -564,26 +564,6 @@ export class Client extends TestHelper {
         test(base.ice_encodingVersion(Ice.Encoding_1_1).ice_getEncodingVersion().equals(Ice.Encoding_1_1));
         test(!base.ice_encodingVersion(Ice.Encoding_1_0).ice_getEncodingVersion().equals(Ice.Encoding_1_1));
```

Comment:
You should also remove the "-2" invocation timeout below. Can wait until a follow-up PR.